plot_equality_data.R

```diff
@@ -52,7 +112,8 @@ columns_to_save = c("mean")
 origin = getwd()
 compare_results = results_table
 for (file in filelist) {
-  load(paste0(origin, file))
+  #load(paste0(origin, file))
+  load("results/res_statdjo_richpoor/Test_raw_scc_SSP3_rcp85_constant_estimates_climensemble_djo.RData")
```

Comment:
I assume this is the leftovers from some working? It obviously doesn't work on my computer as I don't have this file and it makes a mockery of the file options so far.